cmd/clusterctl/OWNERS

```diff
@@ -0,0 +1,11 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+approvers:
+  - sig-cluster-lifecycle-leads
```

Comment:
Since we are not setting `options.no_parent_owners: true`, you can exclude anyone listed as an approver/reviewer for the top level OWNERS file, since it will be inherited by default. https://github.com/kubernetes/community/blob/master/contributors/guide/owners.md#owners

Reply:
While this is true for approvers, I've seen reviewers only be picked from the ones in the list